UploadFile/Page/Pag50410.BCPUpload.al

```diff
@@ -0,0 +1,32 @@
+page 50410 "BCP Upload"
+{
+    PageType = ListPart;
+    ApplicationArea = All;
+    UsageCategory = None;
+    SourceTable = "BCP Upload";
+
+    layout
+    {
+        area(Content)
+        {
+            repeater("File Upload")
+            {
+
+                field("BCP File Name"; Rec."BCP File Name")
+                {
+                    ApplicationArea = All;
+                    ToolTip = 'Filename';
+                    trigger OnDrillDown()
+                    var
+                        downloadFunc: Codeunit "BCP UploadFile";
+                    begin
+                        CurrPage.Update();
```

Comment:
Just question. Why do we need CurrPage.Update() here?

Reply:
Just to ensure the page is up to date and in sync with the data. 
is it unnecessary?

Reply:
I'm not quite sure in what scenarios this would be needed. If you don't know such scenarios either, there is no point in adding it.

Reply:
Noted